natmap/root.js

```diff
@@ -1,9 +1,54 @@
 "use strict";
+const _ = require('lodash');
 const getFromCatalogPath = require('../helpers/getFromCatalogPath');
 const natmap20200903v8 = require('./in/natmap-2020-09-03-v8.json');
 
-// Grab just National Datasets
-const catalog = [getFromCatalogPath(natmap20200903v8, ["National Datasets"])];
+const Agriculture = _.cloneDeep(getFromCatalogPath(natmap20200903v8, ["National Datasets", "Agriculture"]));
+Agriculture.members = Agriculture.members.filter(m => m.name !== "Land Use and Cover in South Australia");
 
-module.exports = Object.assign({}, natmap20200903v8, { catalog: catalog });
+const Communications = _.cloneDeep(getFromCatalogPath(natmap20200903v8, ["National Datasets", "Communications"]));
+Communications.members = Communications.members
+                            .filter(m => m.name !== "ABC Photo Stories (2009-2014)")
+                            .filter(m => m.name !== "ABC Photo Stories by date");
 
+module.exports = {
+  catalog: [
+    {
+      name: "National Datasets",
+      type: "group",
+      members: [
```

Comment:
In the same spirit as Agriculture and Communications you could `getFromCatalogPath` to fetch NationalDatasets, set: 
```
NationalDatasets.members = [Agriculture, Communications];
```

then:
```
return {
  catalog: [NationalDatasets]
};
```